src/js/utils/constants.js

```diff
@@ -1,19 +1,25 @@
 /**
  * Reusable constants
  */
-define([], function() {
+;(function(define) {
     'use strict';
-
-    return {
-        keyCodes: {
-            tab: 9,
-            enter: 13,
-            esc: 27,
-            space: 32,
-            left: 37,
-            up: 38,
-            right: 39,
-            down: 40
-        }
-    };
-});
+    define([], function() {
+        return {
+            keyCodes: {
+                tab: 9,
+                enter: 13,
+                esc: 27,
+                space: 32,
+                left: 37,
+                up: 38,
+                right: 39,
+                down: 40
+            }
+        };
+    });
+}).call(
+    this,
+    (typeof define !== 'undefined' && define) ||
```

Comment:
You shouldn't need the RequireJS check-- I was imagining you would use the exact syntax expected by the optimizer.

We use the namespace property, and the optimizer will rename things accordingly (assuming it is the expected syntax).

http://requirejs.org/docs/faq-advanced.html#rename